apps/judicial-system/backend/src/app/modules/notification/defendantNotification.service.ts

```diff
@@ -152,13 +198,13 @@ export class DefendantNotificationService extends BaseNotificationService {
 
   async sendDefendantNotification(
     type: DefendantNotificationType,
-    defendant: Defendant,
     theCase: Case,
+    defendant: Defendant,
```

Comment:
Just curious, why do you prefer to have the case as the first parameter instead of the defendant? I was thinking about which made more sense when I wrote the service and I felt like since we were in the defendant notification service then it makes sense to focus on the defendant, but maybe it's better to just always have theCase first everywhere to keep things more cohesive

Reply:
Good question. I guess I always think about this as a hierarchy, case includes defendant, includes subpoena, etc. It really does not matter.